Frontend/android-native/app/src/driver/nammaYatriPartner/assets/juspay/v1-configuration.js

```diff
@@ -135,5 +135,12 @@ window.getMerchantConfig = function () {
       "hearingImpairmentVideo" : "https://www.youtube.com/watch?v=udkWOt0serg",
       "genericAccessibilityVideo" : "https://youtu.be/5s21p2rI58c"
     }
+  , "liveStatsDashboard" : {
+    "showLiveStatsDashboard" : true,
+    "liveStatsDashboardUrl" : "https://nammayatri.in/open?source=in-app"
```

Comment:
this variable name can be just `url` and used as `liveStatsDashboard.url` instead of `liveStatsDashboard.liveStatsDashboardUrl`